jboss-marshalling/src/main/java/org/infinispan/jboss/marshalling/io/GridFile.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.io;
+package org.infinispan.jboss.marshalling.io;
```

Comment:
Why was all the GridFile stuff moved? As far as I saw, when I quickly searched, it wasn't using jboss marshalling.

Reply:
Good point. I think I did it because `MarshalledValueByteStream` implemented `org.jboss.marshalling.ByteOutput`, but the move is overkill. Instead I think we can just remove the interface and users (if we have any) will have to call `{@link org.jboss.marshalling.Marshalling#createByteOutput(MarshalledValueByteStream)}` if they require it.

Reply:
Sounds good, should reduce some noise in the PR.